tests/gbs/test_sample.py

```diff
@@ -463,3 +206,16 @@ def test_modes_from_counts():
     modes = [[], [0, 3, 3], [0, 1, 2], [0, 1, 1, 2], [1, 3, 3, 4, 4, 4, 4]]
 
     assert [sample.modes_from_counts(s) for s in counts] == modes
+
+
+def test_postselect():
```

Comment:
Test will need to be updated to reflect postselection on a min and max photon number